namer/namer.cc

```diff
@@ -1261,8 +1261,11 @@ class SymbolDefiner {
             symbolData->flags.isSealed = true;
 
             auto classOfKlass = symbolData->singletonClass(ctx);
-            auto sealedSubclasses =
-                ctx.state.enterMethodSymbol(ctx.locAt(mod.loc), classOfKlass, core::Names::sealedSubclasses());
+            auto loc = ctx.locAt(mod.loc);
+            auto sealedSubclasses = ctx.state.enterMethodSymbol(loc, classOfKlass, core::Names::sealedSubclasses());
+            if (absl::c_count(sealedSubclasses.data(ctx)->locs(), loc) == 0) {
+                sealedSubclasses.data(ctx)->addLoc(ctx, loc);
+            }
```

Comment:
What is the goal with this `c_count` condition?

Reply:
Asking because I'm _pretty_ sure the `c_count` condition is essentially a no-op based on how the `addLoc` method is already implemented.